### DISCUSSION TITLE
etcher not able to verify flashed ev3 image

### DISCUSSION THREAD

**Discussion by jedierikb** [pybricks/support#1265] [Q&A]
on both a mac and windows, using etcher and a just downloaded from LEGO ev3micropythonv200sdcardimage.zip,

I am able to flash a sandisk A1 16gb but fails when checking (see screenshot from mac).
<img width="786" alt="Screenshot 2023-10-24 at 12 35 47 PM" src="https://github.com/pybricks/support/assets/374586/3007c8a5-1f4a-4d7d-9965-6ae64e650a14">

The image, not surprisingly, does not boot an ev3.

Suggestions?

**Comment by dlech** [comment]
This sounds like a defective microSD card. 

Someone also recommended a different flashing program at https://github.com/ev3dev/ev3dev/issues/1596#issuecomment-1757385231.

**Comment by jedierikb** [comment]
Tried a few brand new microSD cards, same problem.

Thank you for the link to the thread on the other flashing program. 

Fixed the problem.

Might want to update the pybricks page that etcher might not work?

**Reply by dlech** [reply]
If just found https://github.com/balena-io/etcher/issues/4079

This suggests that the problem is the ev3dev zip file since flashing the .img inside of it still works. Maybe time to release a new ev3dev image?

**Reply by BertLindeman** [reply]
Naughty: :grin 3.3?
